Mage/src/main/java/mage/abilities/common/CastFromGraveyardOnceStaticAbility.java

```diff
@@ -0,0 +1,107 @@
+package mage.abilities.common;
+
+import mage.MageIdentifier;
+import mage.MageObjectReference;
+import mage.abilities.Ability;
+import mage.abilities.effects.AsThoughEffectImpl;
+import mage.cards.Card;
+import mage.constants.*;
+import mage.filter.FilterCard;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+import mage.watchers.Watcher;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+public class CastFromGraveyardOnceStaticAbility extends SimpleStaticAbility {
```

Comment:
Can you please add a small docstring showing the ability text  that this would go with. I think it would be beneficial for these common abilities to have information like that with them.

Reply:
Not sure what you mean exactly.  Can you give example?  The class wants the full text of the ability.  Currently it's only for Lurrus and RIvaz.  They have the same effect but it's formatted slightly differently.  "During each of your turns, you may cast one" vs "Once during each of your turns, you may cast".

Reply:
What I mean is "CastFromGraveyardOnce" is very broad. One would have to read the implementation or go look at how it's used in order to know what it's for. E.g. to know that it means "Once per turn you may cast a spell from your graveyard".

Reply:
OK, I added a comment.  I thought you were talking about documenting the "text" argument and was a little confused.